domain_generation/riley_hydrate.in

```diff
@@ -0,0 +1,212 @@
+region PERM block INF INF INF INF $(zhi-v_z_delta+10*v_mult1d) $(zhi-10*v_mult1d) units box
+
+# Add water molecules to feed and permeate regions, minimize to move overlapping molecules
+molecule mol1 ../domain_in/tip3p.mol
+create_atoms 0 region FEED subset ${NUM_H20} $(521*v_mult*v_rand) mol mol1 $(322*v_mult*v_rand)
+create_atoms 0 region PERM subset ${NUM_H20} $(512*v_mult*v_rand) mol mol1 $(632*v_mult*v_rand)
+
+lattice hcp ${GRAPHENE_LATTICE_CONST}
+
+
+group FEEDWATER region FEED
+group PERMWATER region PERM
+group WATER type 15 16
+
+timestep 10.0
+minimize         1.0e-8 1.0e-8 1000 100000
+
+# Add 'polysulfone' backing layer to keep membrane in place 
+variable PIN_LOCATION equal $(zhi-v_z_delta+5*v_mult1d)
+region PINlayer block INF INF INF INF ${PIN_LOCATION} $(v_GRAPHENE_LATTICE_CONST+v_PIN_LOCATION) units box
+
+create_atoms 17 region PINlayer
+group PINplane1 type 17
+group PINplane2 region PINlayer
+group PINplane intersect PINplane1 PINplane2
+group PINplane1 delete
+group PINplane2 delete
+
+fix PINFORCE PINplane setforce 0.0 0.0 0.0
+neigh_modify exclude group WATER PINplane
+
+
+# Create 'graphene' pistons to constrain water molecules and simulate operation
+region zwallLO block INF INF INF INF $(zlo) $((zlo)+v_GRAPHENE_LATTICE_CONST*.75) units box
+
+create_atoms 17 region zwallLO 
+group LOzwall1 type 17
+group LOzwall2 region zwallLO
+group LOzwall intersect LOzwall1 LOzwall2
+group LOzwall1 delete
+group LOzwall2 delete
+
+region zwallHI block INF INF INF INF $((zhi)-v_GRAPHENE_LATTICE_CONST*.75) $(zhi) units box
+
+create_atoms 17 region zwallHI
+group HIzwall1 type 17
+group HIzwall2 region zwallHI
+group HIzwall intersect HIzwall1 HIzwall2
+group HIzwall1 delete
+group HIzwall2 delete
+
+group zwalls union LOzwall HIzwall
+group mobile subtract all zwalls
+
+change_box all z delta $(-200*v_mult1d) $(200*v_mult1d) units box
+
+#Pressurize system with pistons
+fix balance1 all balance 0 1.0 shift z 10 1.1
+fix zwalls1 all wall/lj126 zlo $(zlo) 0.06844 3.40700 9 zhi $(zhi) 0.06844 3.40700 9
+fix_modify zwalls1 energy yes
+
+
+variable APPL_PRESS_FEED equal 0.5*${ATM_to_nm2}
+variable APPL_PRESS_PERM equal 0.5*${ATM_to_nm2}
+variable PISTON_AREA equal  ($(xhi)-$(xlo))*($(yhi)-$(ylo))
+variable pos_HYDRATE_FORCE equal ${PISTON_AREA}*${ANG_to_M}*${ANG_to_M}*${APPL_PRESS_FEED}/${FinMD_to_FinN}
+variable neg_HYDRATE_FORCE equal -${PISTON_AREA}*${ANG_to_M}*${ANG_to_M}*${APPL_PRESS_PERM}/${FinMD_to_FinN}
+
+fix FEEDFORCE LOzwall addforce  0.0 0.0 v_pos_HYDRATE_FORCE
+fix PERMFORCE HIzwall addforce  0.0 0.0 v_neg_HYDRATE_FORCE
+
+fix FEEDPISTON LOzwall rigid single torque * off off off force * off off on 
+fix PERMPISTON HIzwall rigid single torque * off off off force * off off on 
+
+unfix PINFORCE  
+unfix balance1
+unfix zwalls1
+# unfix FEEDFORCE
+# unfix PERMFORCE
+# unfix FEEDPISTON
+# unfix PERMPISTON
+# Collect sufficient data for morphological analysis
```

Comment:
Yea I found that commenting out unfix zwalls1 and the ones below it as well make it so that the original error comes back. Adding unfix zwalls1 back changes the error to the lost atoms error.